src/async_fuse/memfs/mod.rs

```diff
@@ -12,7 +12,7 @@ pub mod direntry;
 mod metadata;
 mod node;
 /// Opened files
-mod open_file;
+// pub mod open_file;
```

Comment:
Delete open_file.rs since it's function is replaced, do not leave obsoleted code